src/components/Rocket.js

```diff
@@ -0,0 +1,28 @@
+/* eslint-disable react/prop-types */
```

Comment:
- [ optional ] It will be nice to install `prop-types`  and define the types for each prop